internal/channelz/types.go

```diff
@@ -257,7 +257,7 @@ func (c *channel) deleteSelfFromTree() (deleted bool) {
 // channel, and its memory will be garbage collected.
 //
 // The trace reference count of the channel must be 0 in order to be deleted from the map. This is
-// specified in the channel tracing gRFC that as long as some other trace has reference to an entity,
+// specified in the channel tracing gRPC that as long as some other trace has reference to an entity,
```

Comment:
gRFC is correct here; that's what our proposals are called.

Reply:
Sorry, I will go to restore it.
Thank you.